doctr/models/detection/zoo.py

```diff
@@ -10,25 +10,22 @@
 
 __all__ = ["detection_predictor"]
 
-default_cfgs: Dict[str, Dict[str, Any]] = {
-    'db_resnet50': {'model': 'db_resnet50', 'post_processor': 'DBPostProcessor'},
-    'linknet': {'model': 'linknet', 'post_processor': 'LinkNetPostProcessor'},
-}
+archs = ['db_resnet50', 'linknet']
```

Comment:
if we do this, `archs` is a session constant so it needs to be named in uppercase